fix(docs): small grammar fixes & not correct words (#854)
treeverse/lakeFS: docs/reference/import.md

3. It requires you to switch to using the lakeFS endpoint in all places at once.

## Using lakeFS import tool

> To solve this, we offer an import tool that will not copy any data and allow for a more gradual onboarding process.

Reviewer: Nice.  Mind trying this?
```suggestion
To solve this we offer an import tool that does not copy any data, allowing for a more gradual onboarding process.
```

Author: Yeah, I think we can use that.
Thanks for the suggestion!
